tools/isobuild/bundler.js

```diff
@@ -619,7 +625,9 @@ class File {
         hashes.push(this._inputHash);
       }
 
-      hashes.push(this.sri());
+      if (!this._skipSri) {
```

Comment:
Are you sure this is OK. Because of this change, now effectively when `_skipSri` is set, contents of the file are not part of the hash anymore.